File: app/src/main/java/com/shevart/androidcorelearn/utils/LogUtil.java
package com.shevart.androidcorelearn.utils;

import android.support.annotation.NonNull;
import android.util.Log;

@SuppressWarnings("unused")
public class LogUtil {
    private static final String TAG = "<-- CoreLearnTag -->";

    public static void e(@NonNull String e) {
        Log.e(TAG, e);
    }

    public static void e(@NonNull Throwable e) {
        e.printStackTrace();
    }

    public static void d(@NonNull String e) {
        Log.d(TAG, e);
    }
}

File: app/src/main/java/com/shevart/androidcorelearn/utils/LogUtil.kt
package com.shevart.androidcorelearn.utils

import android.util.Log

object LogUtil {
    private val TAG = "<-- CoreLearnTag -->"

    fun e(e: String) {
        Log.e(TAG, e)
    }

    fun empty() {
        Log.e(TAG, "")
    }

    fun e(e: Throwable) {
        e.printStackTrace()
    }

    fun d(e: String) {
        Log.d(TAG, e)
    }
}
